config/initializers/time_formats.rb

```diff
@@ -5,3 +5,6 @@
 #
 Date::DATE_FORMATS[:stamp] = '%-d %B %Y' # DD MONTH YYYY
 Time::DATE_FORMATS[:stamp] = '%-d %B %Y' # DD MONTH YYYY
+Date::DATE_FORMATS[:short_stamp] = '%-d %b %Y'
+Time::DATE_FORMATS[:short_stamp] = '%-d %b %Y'
```

Comment:
Is this used?

Reply:
yep, https://github.com/ministryofjustice/laa-submit-crime-forms/pull/1011/files#diff-9a2a39ff9660e35e1abafcde500269713f3c08003e54f119144018b827b50d0dR32